makefile.conf.in

```diff
@@ -352,6 +358,10 @@ ifeq ($(ARCH),FREEBSD)
   DYN = -rdynamic
 endif
 
+ifeq ($(BUILDTYPE), WASM)
+    SHARED_LDFLAGS = -Wl
```

Comment:
SHARED_LDFLAGS is set based upon ARCH for most other configurations - if this is truly a unique addition for WASM (and not due to the ARCH), please move this inside the new block at line 284.

Reply:
If I recall correctly (and I might not, since it was two years ago), this started out in the block at line 284, but I moved it further down because the blocks from 326-361 overwrite `SHARED_LDFLAGS`. The actual goal here is just to avoid `-shared`, preferably without repeating a `BUILDTYPE` or `RTLIBTYPE` check within various architecture blocks. Perhaps there's a cleaner way to do it.